### PR TITLE
blog: hydraulics notam

### PR DESCRIPTION
**Pull request by Valastiri**
## Summary of changes
< Provide a summary of the changes in this pull request, ensure that all changes are explained. >
3rd installment of February NOTAMs featuring hydraulics. 

### TODO

- [x] Final Proof
- [x] New Featured Image
- [x] Uploaded Video Media
- [x] Marketing Release

### CODE REVIEW

**Comment by crocket63** [comment]
Soon to come part has already got to master, so I'll try to update this part with the latest stuff we did for hydraulics

File: src/posts/hydraulics-update-2022.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+authors:
+  - 'Davy'
+category: 'UPDATE'
+metaImage: '/img/a32nxwing.png'
+metaAlt: 'FlyByWire Simulations'
+---
+
+## Implemented Features
+
+What’s up in the hydraulics department? Maybe you haven't noticed, but lately some features have already made their way in into the development version.
```
Comment (crocket63) [inline src/posts/hydraulics-update-2022.md:13]: "way in into the"  -> into or in but not both I guess :)

File: src/posts/hydraulics-update-2022.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+*[Can we have Benny hill music for that one? :D]*
+
+---
+
+## Coming Soon to Hydraulics
+
+As we progress, we will simulate all control surfaces in the same realistic manner. Experimental version already has some early versions of elevator and rudder hydraulic simulation, including a realistic left/right elevator split. Don't hesitate to give it a try, and see those surfaces drooping or coming to life depending on hydraulic state of the plane.
+
+Even in with this early version, we are already simulating the fact that in certain speed/pitch demand conditions, the elevator is commanded by all actuators instead of "one slave/one master" actuator combination, so the surface gets the maximum available power to fight against those massive aerodynamic forces. Sadly the surfaces are not receiving any aerodynamic forces due to MSFS SDK limitations, so that control surfaces will even droop in flight if hydraulic power is lost.
```
Comment (crocket63) [inline src/posts/hydraulics-update-2022.md:79]: wether "in" or "with"

File: src/posts/hydraulics-update-2022.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+If it occurs at cruising altitude, you will lose a lot of hydraulic power. Get the plane to a lower altitude and you will see the hydraulics slowly coming back to life. This is already implemented and you can play with these scenarios using the new hydraulic failures available in the EFB menu.
+
+### Emergency Generator
+
+The blue circuit emergency generator is now simulated and will spin at a valve-regulated 12000 rpm to provide electrical power from hydraulic flow. The RAT will provide that flow in the event of an electrical failure and will get you safely on the ground. While there is no 3D RAT model - this will come with the remodel - it has been simulated, with only small features missing (e.g. the anti-stall system,which will come later).
+
+### Physical Based PTU
```
Comment (Benjozork) [inline src/posts/hydraulics-update-2022.md:25]:
```suggestion
### Physics Based PTU
```

**Review by Benjozork — CHANGES_REQUESTED** [review]

**Pull request merged**